HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -84,6 +84,17 @@ def customiseFor16569(process):
 
     return process
 
+def customiseFor17094(process):
+    for mod in ['hltHbhereco','hltHbherecoMethod2L1EGSeeded','hltHbherecoMethod2L1EGUnseeded','hltHfreco','hltHoreco']:
+        if hasattr(process,mod):
+            getattr(process,mod).timeSigmaSiPM = cms.double(2.5)
+            getattr(process,mod).pedSigmaSiPM = cms.double(0.00065)
+            getattr(process,mod).noiseSiPM = cms.double(1)
+            getattr(process,mod).ts4Max = cms.vdouble(100.,45000.)
+            getattr(process,mod).ts4chi2 = cms.vdouble(15.,15.)
+
+    return process
+
```

Comment:
@mtosi 
This customization it will not change the run2-2016 as the second parameters in the config here is not used by the HcalHitReconstructor and the first is identical to what was in previous releases.
For the 2017 the HBHEPhase1Reconstructor  need to be still activated in the hlt sequence